### ISSUE TITLE
customBorders cannot be updated via updateSettings

### ISSUE THREAD

**Issue by leikind** [handsontable/handsontable#2002]
I use `customBorders` while initialising a handsontable, it works perfectly well. But when I use the same data in  `.updateSettings`, it breaks my handsontable. Here is a screencast: http://screencast.com/t/UoOe1VTT

As you see, before I paste the code some cells have white colour, that's thanks to `"customBorders":` in the constructor.

I tried it with 0.11.4 and the latest 0.12.2

Am I missing anything?

Thank you!

**Comment by superRaytin** [comment]
Same problem here.

**Comment by AMBudnik** [comment]
Please prepare a [JSFiddle](https://jsfiddle.net/api/post/library/pure/) and we'll see what we can do.

**Comment by phiresky** [comment]
Check this example, which adds borders after 1 second. https://jsfiddle.net/Lkdyvjqm/ (adapted from the docs example)

It works when you manually call runHooks('afterInit')

**Comment by AMBudnik** [comment]
great demo @phiresky thanks for support.

**Comment by phiresky** [comment]
By the way `runHooks(afterInit)` doesn't really fix it, you can only add borders that way, removing them does nothing

**Comment by AMBudnik** [comment]
Confirmed with 0.17.0 that `customBorders` does not change with `updateSettings`

**Comment by AMBudnik** [comment]
Confirmed with 0.24.0, demo: http://jsfiddle.net/handsoncode/uygpwyod/2/

**Comment by jaredbaszler** [comment]
Add me as one who would like to see this feature added.

**Comment by abhirakshit** [comment]
Any updates on this please?

**Comment by AMBudnik** [comment]
Hi @abhirakshit 
Currently not, but this issue will probably be fixed with a `updateSettings` refactor.

**Comment by ssbunny** [comment]
Confirmed with 0.28.3 that `customBorders` does not change with `updateSettings` .

**Comment by elarandjelovic** [comment]
I've run into this problem too, any updates yet? Thanks :)

**Comment by AMBudnik** [comment]
Hi guys,

The `updateSettings` will be refactored as some crucial options do not change while using it. However, we do not have any make-it-work date planned. Please check our Roadmap to be updated about new tasks https://trello.com/b/PztR4hpj/handsontable-roadmap-2017-login-to-vote
You can also leave your vote (for logged accounts) for favorite or a needed fix or a feature.

**Comment by elarandjelovic** [comment]
Great stuff, thanks.  Which card on the Trello board does this come under so I can add my vote?

**Comment by AMBudnik** [comment]
It is here https://trello.com/c/aolvzuji/67-release-handsontable-community-edition-1-0-0-q1-q3
as a part of a Hot CE 1.0.0 project.

**Comment by zwei8908** [comment]
Is there any way to remove custom border?

**Comment by AMBudnik** [comment]
Hi @zwei8908 

The updateSettings method is buggy (as in the title). But the border can be erased via context menu (user action). I do not see any convenient way to do it via setCellMeta programmatically. 
I guess that we need to add some methods that will allow developers to easily change and erase borders.

**Comment by AMBudnik** [comment]
The issue is no longer replicable in v `2.0.0`, where we do not need to call `afterInit` hook https://jsfiddle.net/yswnztg0/

**Comment by zippeellii** [comment]
I still experience this being a problem when applying multiple borders to the same cell or cellRange in different calls to updateSettings. For instance, first adding a left border and then in a later stage adding a right border. I slightly modified the fiddle adding more borders after 3000 ms. https://jsfiddle.net/13pdLn2u/9/

I solved it by accessing the CustomBorders plugin and run the `resetBorders()` method. (Line 40).

Figured this information might be helpful to someone.
